lms/services/canvas_api/client.py

```diff
@@ -40,14 +43,22 @@ class CanvasAPIClient:
         Canvas API request fails for any other reason
     """
 
-    def __init__(self, authenticated_client, file_service):
+    def __init__(
+        self,
+        authenticated_client,
+        file_service: FileService,
+        with_folders: bool = False,
```

Comment:
Again for consistency with my suggested application instance and JS config setting names:

```suggestion
        folders_enabled: bool = False,
```